spec/requests/v1/songs_request_spec.rb

```diff
@@ -11,3 +11,19 @@
     end
   end
 end
+
+describe 'song endpoint' do
+  describe 'GET /song' do
```

Comment:
The endpoint for the `songs#show` action is actually `GET /songs/:id`, so I'd reference that here instead of `GET /song` (which actually won't go anywhere). :)